social-circus/src/config/RouteConfig.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Route, Routes } from "react-router-dom";
+import { Error, Explore, Feed, Home, Login, Profile, Signup } from "../routes";
+
+export const RouteConfig = () => {
+  return (
+    <Routes>
+      <Route path="/login" element={<Login />} />
+      <Route path="/signup" element={<Signup />} />
+      <Route path="/explore" element={<Explore />} />
+      <Route path="/feed" element={<Feed />} />
```

Comment:
you can omit `/` on the paths in the beginning other than the home path.  Not needed in React router v6

path='feed' will work

Reply:
Oh! Wasn't aware of this. Thanks!